examples/panel/stock-market-chatbot/chat.py

```diff
@@ -0,0 +1,35 @@
+from openai import OpenAI
+import os
+from dotenv import load_dotenv
+import pandas as pd
+
+load_dotenv(".env")
```

Comment:
remove

Reply:
i guess this was addressed ?

Reply:
Resolved 


[here ](https://github.com/ploomber/doc/pull/124/commits/5134a06dd9966b18ae5cc687803e19a4ddf4f0c9)